plugin/src/main/java/io/snyk/languageserver/protocolextension/messageObjects/scanResults/SnykScanParam.java

```diff
@@ -0,0 +1,45 @@
+package io.snyk.languageserver.protocolextension.messageObjects.scanResults;
+
+public class SnykScanParam {
```

Comment:
I think this class needs `equals` and `hashCode` methods to be able to sort & compare them